testSrc/com/koxudaxi/pydantic/PydanticCompletionTest.kt

```diff
@@ -33,12 +33,10 @@ open class PydanticCompletionTest : PydanticTestCase() {
             "Annotated",
             "MISSING",
             "WindowsError")
-        val actual = myFixture!!.completeBasic().filter {
-            it!!.psiElement is PyTargetExpression
-        }.filterNot {
-            excludes.contains(it!!.lookupString)
+        val actual = (myFixture!!.completeBasic() ?: emptyArray()).filter {
```

Comment:
`completeBasic` can return `null`, so I added a default value, this made one of the tests pass.